packages/cdktf-cli/templates/python-pip/.hooks.sscaff.js

```diff
@@ -32,7 +32,11 @@ exports.post = options => {
   }
 
   writeFileSync('requirements.txt', pypi_cdktf, 'utf-8');
-  execSync('pip3 install --user -r requirements.txt', { stdio: 'inherit' });
+  installArgs = '';
```

Comment:
```suggestion
  let installArgs = '';
```
Let's keep it a local variable, JavaScript works a bit different than Python here 😅

Reply:
Sure thing.  Working in too many different languages, hard to remember all the syntax differences...

Reply:
Yeah, I thought so 😁 Been there as well